apps/civicsignalblog/server.ts

```diff
@@ -14,7 +13,16 @@ loadEnvConfig(projectDir, dev);
 
 const hostname = process.env.NEXT_HOSTNAME || "localhost";
 const port = parseInt(process.env.PORT || "3000", 10);
-const sendGridAPIKey = process.env.SENDGRID_API_KEY;
+const smtpAuthPass = process.env.SMTP_PASS || process.env.SENDGRID_API_KEY;
+const smtpFromName =
+  process.env.SMTP_FROM_NAME ||
+  process.env.SMTP_FROM_NAME ||
+  "Civic Signal Blog CMS";
```

Comment:
```suggestion
  process.env.SMTP_FROM_NAME ||
  process.env.SENDGRID_FROM_NAME ||
  process.env.NEXT_PUBLIC_APP_NAME ||
  "Civic Signal Blog CMS";
```

`SMTP_FROM_NAME` is repeated.